core/src/test/scala/unit/kafka/admin/ResetConsumerGroupOffsetTest.scala

```diff
@@ -174,24 +178,31 @@ class ResetConsumerGroupOffsetTest extends KafkaServerTestHarness {
           val sumOffset = (assignments.filter(_.topic.exists(_ == topic1))
             .filter(_.offset.isDefined)
             .map(assignment => assignment.offset.get) foldLeft 0.toLong)(_ + _)
-          sumOffset == 100
+          sumOffset == 150
         case _ => false
       }
     }, "Expected that consumer group has consumed all messages from topic/partition.")
 
     executor.shutdown()
 
-    val cgcArgs1 = Array("--bootstrap-server", brokerList, "--reset-offsets", "--group", group, "--all-topics", "--to-datetime", format.format(checkpoint), "--execute")
+    val cgcArgs1 = Array("--bootstrap-server", brokerList, "--reset-offsets", "--group", group, "--all-topics", "--to-datetime", format1.format(checkpoint1), "--execute")
     val opts1 = new ConsumerGroupCommandOptions(cgcArgs1)
     val consumerGroupCommand1 = createConsumerGroupService(opts1)
 
     TestUtils.waitUntilTrue(() => {
       val assignmentsToReset = consumerGroupCommand1.resetOffsets()
-      assignmentsToReset.exists { assignment => assignment._2.offset() == 50 }
+      assignmentsToReset.exists { assignment => assignment._2.offset() == 100 }
     }, "Expected the consumer group to reset to when offset was 50.")
 
     printConsumerGroup()
 
+    // test old format. don't do validity check, since it wont work on some timezones
+    val format = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSSX")
+    val cgcArgs2 = Array("--bootstrap-server", brokerList, "--reset-offsets", "--group", group, "--all-topics", "--to-datetime", format.format(checkpoint), "--execute")
+    val opts2 = new ConsumerGroupCommandOptions(cgcArgs2)
+    val consumerGroupCommand2 = createConsumerGroupService(opts2)
+    consumerGroupCommand2.resetOffsets()
```

Comment:
Not sure what we're actually validating here. Maybe we could just test the `getDateTime` function directly in a separate test case to ensure compatibility?